package.json

```diff
@@ -82,6 +85,7 @@
     "@deltachat/react-qr-reader": "^4.0.0",
     "@mapbox/geojson-extent": "^1.0.0",
     "application-config": "^1.0.1",
+    "arg": "^5.0.2",
```

Comment:
does not look like this is still needed

Reply:
oh yeah, I forgot to remove it, I'll change it